packages/web/src/executors/webpack/schema.json

```diff
@@ -89,7 +93,8 @@
       "default": [],
       "items": {
         "$ref": "#/definitions/assetPattern"
-      }
+      },
+      "x-priority": "important"
```

Comment:
There's 8 important items in here, which is a bit too much. I know the artificial limit we set was 7, but I think less is more here. 
So maybe find some more places to remove the important flag, maybe `vendorChunk` or `styles`?